Lib/idlelib/idle_test/mock_tk.py

```diff
@@ -79,7 +79,7 @@ def tearDownClass(cls):
     ---
     For 'ask' functions, set func.result return value before calling the method
     that uses the message function. When messagebox functions are the
-    only gui alls in a method, this replacement makes the method gui-free,
+    only gui calls in a method, this replacement makes the method gui-free,
```

Comment:
These are my errors.  GUI is standard for this acronym and I now use it.
```suggestion
    only GUI calls in a method, this replacement makes the method GUI-free,
```